source/opt/mem_pass.cpp

```diff
@@ -601,12 +610,15 @@ Pass::Status MemPass::InsertPhiInstructions(ir::Function* func) {
           // Replace load's id with the last stored value id for variable
           // and delete load. Kill any names or decorates using id before
           // replacing to prevent incorrect replacement in those instructions.
-          const uint32_t loadId = ii->result_id();
+          const uint32_t loadId = inst->result_id();
           context()->KillNamesAndDecorates(loadId);
           (void)context()->ReplaceAllUsesWith(loadId, replId);
-          context()->KillInst(&*ii);
+          context()->KillInst(inst);
         } break;
-        default: { } break; }
+        default:
+          break;
+      }
```

Comment:
Oh, yeah, thanks for this one.  Ran into it the other day and I forgot to fix it.